ipwb/util.py

```diff
@@ -26,7 +26,7 @@
 from ipfsapi.exceptions import ConnectionError
 
 
-IPFSAPI_IP = '127.0.0.1'
+IPFSAPI_IP = 'localhost'
```

Comment:
Using `localhost` here is equally as valid as `127.0.0.1`. This means, while this change is supposed to work as expected, this is not a **REQUIRED** change for this purpose.

Reply:
localhost makes it consistent with IPWBREPLAY_IP, which is now a bit of a misnomer.

Thanks for the quick feedback.